### ISSUE TITLE
Continue cventry on next page, rather than break to next page

### ISSUE THREAD

**Issue by gmavrom** [posquit0/Awesome-CV#261]
Dear all,

I need your help fixing one minor issue with the "Awesome-CV" template.

That is when a new \cventry is introduced, which is close to the end of the page and contains many cvitems, then the whole entry is moved to the next page, rather than just the bullet points continuing on.

I am confident that this is because of the \cventry being basically a "tabular*", which means it cannot continue to the next page. 

A potential solution to that is to rewrite the \cventry with longtable rather than with tabular*. I have tried to do that but with no desirable results so far...

Anyone more brave than me is willing to give this a go? Or perhaps suggest another possible solution?

**Comment by cfriedline** [comment]
You've probably figured this out by now, but I deal with this by creating two different `cventries`, the first contains the org, job title, dates, location and one set of `cvitems`. The following one just contains the overflow of the cv items, which I do by eye. Essentially.

```
\cventry
    {} % Organization
    {} % Job title
    {} % Date(s)
    {} % Location
    {
        \begin{cvitems}
```

**Comment by gmavrom** [comment]
Yeah, I ended up doing the same essentially. The problem is that I am somewhat obsessed with having the same exact distances between items of the same objects. So by using the approach you recommend, I have to play a lot with \vspace{} commands in order to make it look identical!

Thanks a lot! :)

**Comment by Judoludo** [comment]
hi there! it's been 2 years since the OP, but I'd need the solution to this exact same problem. @gmavrom did you fix it?? or anyone else? please let me know, you would save my life :) thank you!!

**Comment by Judoludo** [comment]
> Dear all,
> 
> I need your help fixing one minor issue with the "Awesome-CV" template.
> 
> That is when a new \cventry is introduced, which is close to the end of the page and contains many cvitems, then the whole entry is moved to the next page, rather than just the bullet points continuing on.
> 
> I am confident that this is because of the \cventry being basically a "tabular*", which means it cannot continue to the next page.
> 
> A potential solution to that is to rewrite the \cventry with longtable rather than with tabular*. I have tried to do that but with no desirable results so far...
> 
> Anyone more brave than me is willing to give this a go? Or perhaps suggest another possible solution?

hi there! it's been 2 years since the OP, but I'd need the solution to this exact same problem. @gmavrom did you fix it?? or anyone else? please let me know, you would save my life :) thank you!!

**Comment by Sed9yDataBank** [comment]
lets say for example this:

```latex
%---------------------------------------------------------
  \cventry
    {Role} % Job title
    {Company} % Organization
    {United States} % Location
    {2016 - 2017} % Date(s)
    {
      \begin{cvitems} % Description(s) of tasks/responsibilities
        \item {ONE}
        \item {TWO}
        \item {THREE}
        \item {FOUR}
        \item {FIVE}
        \item {SIX}
        \item {SEVEN}
      \end{cvitems}
    }
 
%---------------------------------------------------------
```

and your first page is not enough to fill all of the above so the whole work experience bullet points are automatically moved to the second page, which will create a huge white space in your first page.

the solution would be to do the following:

```latex
%---------------------------------------------------------
  \cventry
    {Role} % Job title
    {Company} % Organization
    {United States} % Location
    {2016 - 2017} % Date(s)
    {
      \begin{cvitems} % Description(s) of tasks/responsibilities
        \item {ONE}
        \item {TWO}
        \item {THREE}
        \item {FOUR}
      \end{cvitems}
    }
 
%---------------------------------------------------------
  \cventry
    {} % Job title
    {} % Organization
    {} % Location
    {} % Date(s)
    {
      \begin{cvitems} % Description(s) of tasks/responsibilities
        \item {FIVE}
        \item {SIX}
        \item {SEVEN}
    }
    
%---------------------------------------------------------
```

**Comment by TheSkay** [comment]
Dear all,

<ins>There's still no solution other than the workaround?</ins>

I started using Awesome CV almost 5 years ago and added settings allowing certain experiences to be displayed or hidden.

Therefore the workaround does not fit my case.

Is there ever a plan to resolve this problem?

Thank you.
